app/Http/Controllers/PostController.php

```diff
@@ -91,17 +93,18 @@ public function getAllPost(Request $request)
         	? $this->post
         	: $this->post->where('offer_or_ask',$request->offer_or_ask);
 
+        $count          = $posts->count();
         $postAfterSkip  = $posts->skip($offset)
             ->take(config('constants.POST_SIZE'))->get();
 
         ImageHelper::includeImageUserInPostResponse($postAfterSkip);
```

Comment:
I don't support the concept of facade. Create a class and access the contents instead.